mmv1/products/dialogflowcx/Agent.yaml

```diff
@@ -111,3 +114,96 @@ properties:
     name: 'enableSpellCorrection'
     description: |
       Indicates if automatic spell correction is enabled in detect intent requests.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'advancedSettings'
```

Comment:
Note that we'll still need this block to be `default_from_api`, because it can get returned by the API when nothing has been set.

Reply:
Also `advancedSettings.loggingSettings.enableStackdriverLogging` is expected to be `true` if `Agent.enableStackdriverLogging` is `true`, we won't even have an empty object, but an object with fields we're ignoring 😅.

Reply:
(I tested it and it worked locally, so I'm going to resolve this conversation; thanks!)